spec/dummy/app/models/series.rb

```diff
@@ -0,0 +1,2 @@
+class Series < ActiveRecord::Base
+end
```

Comment:
Could you please add a new line at the end of the file? I updated the Rubocop settings to also include the dummy app, so would have been catched by HoundCI. Thanks